src/quo2/components/page_nav.cljs

```diff
@@ -0,0 +1,216 @@
+(ns quo2.components.page-nav
+  (:require [clojure.string :as string]
+            [quo.react-native :as rn]
+            [quo.theme :as theme]
+            [quo2.foundations.colors :as colors]
+            [status-im.ui.components.icons.icons :as icons]
+            [status-im.utils.dimensions :as dimensions]))
+
+(def centrify
+  {:display :flex
+   :justify-content :center
+   :align-items :center})
+
+(def align-left
+  (merge centrify
+         {:align-items :flex-start}))
+
+(def icon-styles (merge
+                  centrify
+                  {:width 32
+                   :height 32
+                   :border-radius 10}))
+
+(defn mid-section
+  [{:keys [mid-section-type left-align? mid-section-description mid-section-description-user-icon mid-section-description-icon mid-section-main-text mid-section-right-icon mid-section-icon mid-section-main-text-icon-color mid-section-left-icon] :or {left-align? false}}]
+  (let [text-color (if (theme/dark?)
+                     colors/white
+                     colors/black)
+        text-secondary-color (if (theme/dark?)
+                               colors/neutral-40
+                               colors/neutral-50)]
+    [rn/view {:id "mid"
+              :style (merge
+                      (if left-align?
+                        align-left
+                        centrify)
+                      {:flex 1})}
+     (case mid-section-type
+       :text-only [rn/text {:style {:color text-color}} mid-section-main-text]
+       :text-with-two-icons [rn/view (assoc
+                                      centrify
+                                      :flex-direction :row
+                                      :width 32
+                                      :height 32)
+                             [icons/icon mid-section-left-icon
+                              {:width 20
+                               :height 20
+                               :color mid-section-main-text-icon-color}]
+                             [rn/text {:style
```

Comment:
you should use `weight` and `size` props instead

Reply:
in all text components

Reply:
Done